presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlDistributedQueries.java

```diff
@@ -78,6 +79,24 @@ public void testViews()
         execute("DROP VIEW IF EXISTS tpch.test_view");
     }
 
+    @Test
+    public void testSelectNativeDataTypeColumn()
+            throws Exception
+    {
+        execute("CREATE TABLE tpch.test_other_data_type (_foo UUID)");
```

Comment:
To be more standardized, please use `assertUpdate`, similarly to `TestMySqlIntegrationSmokeTest`.